token/core/cmd/pp/common/common.go

```diff
@@ -68,3 +86,60 @@ func SetupIssuersAndAuditors(pp PP, Auditors, Issuers []string) error {
 	}
 	return nil
 }
+
+func ReadFile(file string) ([]byte, error) {
+	fileCont, err := ioutil.ReadFile(file)
+	if err != nil {
+		return nil, errors.Wrapf(err, "could not read file %s", file)
+	}
+
+	return fileCont, nil
+}
+
+func ReadPemFile(file string) ([]byte, error) {
+	bytes, err := ReadFile(file)
+	if err != nil {
+		return nil, errors.Wrapf(err, "reading from file %s failed", file)
+	}
+
+	b, rest := pem.Decode(bytes)
+	if b == nil {
+		return nil, errors.Errorf("no pem content for file %s", file)
+	}
+	if len(rest) != 0 {
+		return nil, errors.Errorf("extra content after pem file %s", file)
+	}
```

Comment:
We could just ignore the rest. If there are more certs in the file, we don't care, do we?

Reply:
I would say, let's constraint to a single certificate in the file